go/merge/three_way.go

```diff
@@ -11,7 +11,19 @@ import (
 	"github.com/attic-labs/noms/go/types"
 )
 
-// ErrMergeConflict indicates that a merge attempt failed and must be resolved manually for the provided reason.
+// ResolveFunc is the type for custom merge-conflict resolution callbacks.
+// When the merge algorithm encounters two non-mergeable changes (aChange and
+// bChange) at the same path, it calls the ResolveFunc passed into ThreeWay().
+// The callback gets the types of the two incompatible changes (added, changed
+// or removed) and the two Values that could not be merged (if any). If the
```

Comment:
Would it be correct to say "two keys" rather than "two Values"? Because this Value is different to the one referenced below "the new value to be used (if any)" and it would be helpful to disambiguate.

And if so - perhaps instead of saying "the new value to be used (if any)" say "the new value for that key (if any)"?